Source/ASDisplayNode.mm

```diff
@@ -742,6 +744,8 @@ - (UIView *)view
 
 - (CALayer *)layer
```

Comment:
I think we have to add locking to this method ... any reason why we didn't have in the past?